base/cvd/cuttlefish/host/commands/cvd/server_command/utils.h

```diff
@@ -84,15 +84,21 @@ Result<cvd::Response> NoGroupResponse(const RequestWithStdio& request);
 // not sufficients to choose one from. The function does not verify that.
 Result<cvd::Response> NoTTYResponse(const RequestWithStdio& request);
 
-enum class TerminalColors : int {
-  kReset = 0,
-  kBoldRed = 1,
-  kCyan = 2,
-  kRed = 3,
+class TerminalColors {
+ public:
+  TerminalColors(bool is_tty);
+  const std::string& Reset() const { return reset_; }
+  const std::string& BoldRed() const { return bold_red_; }
+  const std::string& Red() const { return red_; }
+  const std::string& Cyan() const { return cyan_; }
+
+ private:
+  const std::string reset_;
+  const std::string bold_red_;
+  const std::string red_;
+  const std::string cyan_;
```

Comment:
Rather than making these all instance state, can `is_tty` be the only instance state and the getter methods return `std::string_view`s based on the constants in `utils.cpp`?

Reply:
Done